linera-core/src/client/mod.rs

```diff
@@ -296,11 +296,13 @@ impl<Env: Environment> Client<Env> {
         match self.local_node.chain_info(chain_id).await {
             Ok(info) => Ok(info),
             Err(LocalNodeError::BlobsNotFound(blob_ids)) => {
+                // If the chain is missing then the error is a WorkerError
+                // and so a BlobsNotFound
                 // TODO(#2351): make sure the blobs are legitimate!
```

Comment:
@afck What is this TODO? Do we a pending security bug here?

Reply:
Yes, the same one that we talked about: The client downloads the blobs, writes them to storage and initializes chains without checking that those blobs were actually published by another chain, i.e. that the chain was actually ever opened.